### PR TITLE
Lite kafka event timestamp union

### PR DESCRIPTION
**Pull request by maciej-brzezinski**

### CODE REVIEW

File: api/src/main/scala/pl/touk/nussknacker/engine/api/Context.scala [hunk 1]
```diff
@@ -64,4 +64,7 @@ case class Context(id: String, variables: Map[String, Any], parentContext: Optio
   def popContext : Context =
     parentContext.getOrElse(throw new RuntimeException("No parent context available"))
 
+  def clearRedundantVariablesExcept(variablesToLeave: Set[String]) : Context =
```
Comment (mproch) [inline api/src/main/scala/pl/touk/nussknacker/engine/api/Context.scala:67]: Can we have a method like `clearUserVariables`? or sth like that? And have EventTimestampVariableName in that method?

Comment (maciej-brzezinski) [reply api/src/main/scala/pl/touk/nussknacker/engine/api/Context.scala]: changed it, please check if it's okay now

Comment (mproch) [reply api/src/main/scala/pl/touk/nussknacker/engine/api/Context.scala]: Let's add a comment that it leaves technical variables, hidden from user

**Review by mproch — APPROVED** [review]
:+1: let's just add comment to Context method

**Pull request merged**